src/css/main.scss

```diff
@@ -1122,6 +1126,18 @@ div.search-input:focus-within {
   }
 }
 
+.action-dots-enabled {
+
+  &:hover {
+    background-color: var(--blue-primary);
+    color: #FFFFFF;
+  }
+}
+
+.action-dots-disabled {
+  cursor: auto !important;
+}
+
```

Comment:
As I mentioned before, this should probably be handled with `:disabled` sudo class instead of separate classes.

Additionally I think that we should add styling `opacity: 0.5` to the button when it is disabled and maybe remove the hover effect (currently action-dots button has blue background on hover). This way user will not be confused why the dropdown is not appearing when clicking the action-dots button

Reply:
> (currently action-dots button has blue background on hover)

I removed this effect by using the `action-dots-enabled` class. After changing it as you suggested I just used `pointer-events: none;` when the button is disabled